src/modules/logText.js

```diff
@@ -7,32 +7,28 @@ const conditions = [
     msg => msg.includes(' '),
 ];
 
-module.exports = async function (response, { input, db, id }) {
+module.exports = async function logText(response, context) {
+    const { input, setModuleData, getModuleData } = context;
     const msgFit = reduce(
         conditions,
         (acc, condition) => condition(input),
         true,
     );
 
-    // TODO one request/query
-    const updateQuery = {
-        $inc: {
-            'data.log.allCounter': 1,
-        },
-    };
+    const { allCounter, fitCounter } = await getModuleData('log', context);
```

Comment:
now it sync call, but in future getModuleData can be async... or i dont know its look better with awat! Why u ask me? why? oh...